aasemble/django/apps/api/tests.py

```diff
@@ -458,6 +458,27 @@ def test_refresh_mirror(self, refresh_mirror):
         self.client.post(mirror['self'] + 'refresh/')
         self.assertTrue(refresh_mirror.delay.call_args_list)
 
+    def test_get_correct_mirror_for_user(self):
+        self.test_create_mirror()
+
+        data = {'url': 'http://example2.com/',
+                'series': ['trusty'],
+                'components': ['main']}
+        authenticate(self.client, 'aaron')
+        self.client.post(self.list_url, data, format='json')
+        response = self.client.get(self.list_url)
+        self.assertEquals(len(response.data['results']), 1, 'did not return only 1 mirror')
+        self.assertEquals(response.data['results'][0]['url'], 'http://example2.com/', 'url not the same as created')
+        self.assertNotEquals(response.data['results'][0]['url'], 'http://example.com/', 'url same as previous url')
```

Comment:
This last test can't possibly fail if the preceding one passes, so it's superfluous.